service/controller/chartconfig/v7/key/key.go

```diff
@@ -7,6 +7,16 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
+const (
+	// CordonReasonAnnotationName is the name of the annotation that indicates
+	// the reason of why chart-operator should not apply any update on this chart CR.
+	CordonReasonAnnotationName = "app-operator.giantswarm.io/cordon-reason"
+
+	// CordonUntilAnnotationName is the name of the annotation that indicates
+	// the expiration date of rule of this cordon.
+	CordonUntilAnnotationName = "app-operator.giantswarm.io/cordon-until"
```

Comment:
Let's have an `pkg/annotation` package like we already have in `aws-operator` and `cluster-operator`. We can then import the annotations like `annotation.CordonUntil`.